_layouts/reveal.html

```diff
@@ -62,13 +62,22 @@
 								| replace:'<backgroundimageopacity>','<!-- .slide: data-background-opacity="'
 								| replace:'</backgroundimageopacity>','" -->'
 %}{%
-							assign first_char = line | strip
```

Comment:
Was it important to remove the `strip` filter? I *think* this is there to make sure we still detect indented fragments. Looking at your change locally it looks like the indented fragment now renders with it’s parent rather than as a separate step.